expat/tests/xmltest.log.expected

```diff
@@ -6,5 +6,5 @@ Expected not well-formed: sun/not-wf/uri01.xml
 Expected not well-formed: oasis/p06fail1.xml
 Expected not well-formed: oasis/p08fail1.xml
 Expected not well-formed: oasis/p08fail2.xml
-Passed: 1801
-Failed: 8
+Passed: 1800
+Failed: 9
```

Comment:
That looks like we have a single failed test more now? Please share the story with this.